ui/resizable.js

```diff
@@ -994,8 +997,8 @@ $.ui.plugin.add("resizable", "alsoResize", {
 				$(exp).each(function() {
 					var el = $(this);
 					el.data("ui-resizable-alsoresize", {
-						width: parseInt(el.width(), 10), height: parseInt(el.height(), 10),
-						left: parseInt(el.css("left"), 10), top: parseInt(el.css("top"), 10)
+					    width: parseInt(el.css("width"), 10), height: parseInt(el.css("height"), 10),
```

Comment:
I put some notes together on width() vs css('width') vs outerWidth() here:
http://jsfiddle.net/ilancopelyn/n911jymr/1/
Have a play around with it and see what you think - there are some subtleties with trying to be consistent.